### PR TITLE
Merge the "jenkins-aws" branch to master

### PR DESCRIPTION
**Pull request by jonesmz**
Automated builds work now

### CODE REVIEW

File: parsers/addsets/addsets [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/addsets/addsets:1]: I'm not a fan of keeping the `-enx` for bash.

File: parsers/binaryWorkarounds/binaryWorkarounds [hunk 2]
```diff
@@ -1,11 +1,11 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/binaryWorkarounds/binaryWorkarounds:1]: I'm not a fan of keeping the `-enx` for bash.

File: parsers/compress-image/compress-image [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/compress-image/compress-image:1]: Not a fan

File: parsers/convertimg/convertimg [hunk 4]
```diff
@@ -1,3 +1,3 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/convertimg/convertimg:1]: Not a fan

File: parsers/distcc/distcc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/distcc/distcc:1]: Not a fan

File: parsers/downloadstage3/downloadstage3 [hunk 6]
```diff
@@ -1,24 +1,22 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/downloadstage3/downloadstage3:1]: Not a fan, but probably worth keeping.

File: parsers/echo/echo [hunk 7]
```diff
@@ -1,2 +1,2 @@
-#!/bin/bash
-echo $@
\ No newline at end of file
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/echo/echo:1]: Not a fan

File: parsers/git/git [hunk 8]
```diff
@@ -1,18 +1,50 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/git/git:1]: Not a fan

File: parsers/mdadm-shutdown/mdadm-shutdown [hunk 9]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/mdadm-shutdown/mdadm-shutdown:1]: Not a fan

File: parsers/packstage4/packstage4 [hunk 10]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/packstage4/packstage4:1]: Not a fan

File: parsers/password/password [hunk 11]
```diff
@@ -1,9 +1,14 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/password/password:1]: Not a fan

File: parsers/printf/printf [hunk 12]
```diff
@@ -1,2 +1,2 @@
-#!/bin/bash
-printf $@
\ No newline at end of file
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/printf/printf:1]: Not a fan

File: parsers/qemu/qemu [hunk 13]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/qemu/qemu:1]: Not a fan

File: parsers/rawcommand/rawcommand [hunk 14]
```diff
@@ -1,14 +1,20 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/rawcommand/rawcommand:1]: Not a fan

File: parsers/setprofile/setprofile [hunk 15]
```diff
@@ -1,11 +1,9 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/setprofile/setprofile:1]: Not a fan

File: parsers/setupservices/setupservices [hunk 16]
```diff
@@ -1,8 +1,14 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/setupservices/setupservices:1]: Not a fan

File: parsers/stage3/stage3 [hunk 17]
```diff
@@ -1,5 +1,4 @@
-#!/bin/bash -e
-
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/stage3/stage3:1]: Not a fan

File: parsers/strip/strip [hunk 18]
```diff
@@ -1,13 +1,17 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/strip/strip:1]: Not a fan

File: parsers/unmount/unmount [hunk 19]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/unmount/unmount:1]: Not a fan

File: parsers/unpackstage4/unpackstage4 [hunk 20]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/unpackstage4/unpackstage4:1]: Not a fan

File: parsers/users/setupusers [hunk 21]
```diff
@@ -1,21 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -evx
```
Comment (samip5) [inline parsers/users/setupusers:2]: Not a fan

File: parsers/users/users [hunk 22]
```diff
@@ -1,11 +1,12 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/users/users:1]: Not a fan

File: parsers/utils/utils [hunk 23]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -evx
```
Comment (samip5) [inline parsers/utils/utils:1]: Not a fan

**Review by samip5 — CHANGES_REQUESTED** [review]
I have reviewed what I think of it's state. I also think that the genpi-releaser thing is not ready to be merged nor the fact that builds get null prefix.

File: parsers/addsets/addsets (hunk 1 above)
Comment (jonesmz) [reply parsers/addsets/addsets:1]: What's the problem with it?

Comment (samip5) [reply parsers/addsets/addsets:1]: > What's the problem with it?

I dislike the whole lot of things it outputs, specially if ran manually.

**Comment by samip5** [comment]
Also, they may work but some stages are still failing but are not outputting why.

**Comment by jonesmz** [comment]
So you just want the -vx removed?

**Comment by samip5** [comment]
> So you just want the -vx removed?

Yes.

**Review by samip5 — APPROVED** [review]
LGTM now.

**Pull request merged**